src/transformers/modeling_tf_bert.py

```diff
@@ -838,25 +903,34 @@ def call(self, inputs, **kwargs):
             input_ids = tf.constant(tokenizer.encode("Hello, my dog is cute", add_special_tokens=True))[None, :]  # Batch size 1
             outputs = model(input_ids)
             prediction_scores, seq_relationship_scores = outputs[:2]
-
         """
-        return_dict = kwargs.get("return_dict")
-        return_dict = return_dict if return_dict is not None else self.bert.return_dict
```

Comment:
This breaks the current behavior where not passing `return_dict` defaults to the value in the config. This line should be added back.

Reply:
I don't see where is the issue by removing this line, do you have a breaking example in mind?

But I can put it back, no problem.

Reply:
It's what I said: in most cases, you don't pass `return_dict=True` when calling the model, it's set by the `config` at initialization, for instance when doing:
```
model = TFBertModel.from_pretrained('bert-base-uncased', return_dict=True)
```
This line looks what the value in the config passed at init (stored in `self.bert`) was if `return_dict` is not passed.

Reply:
Yes, if you set `return_dict=True` when you create your model, it will be always True unless you set yourself `return_dict=False` when you call your model. Is that not the expected behavior?

Reply:
It is, but if you remove this line, `return_dict` will be None when you test it later to decide which output to return, so you won't return the dict-like outputs.

Reply:
I just tried many combination I could think about and I always get the expected behavior, I don't see how `return_dict` can be None if it is set in the model call.

Reply:
But I'm doing a revert it is ok no worries :)

Reply:
>  I don't see how return_dict can be None if it is set to True or False in the model call.

That is the point, in most cases the user does not pass the `return_dict` argument to the model because it has been set at init time via the config (same for `output_attentions` and `output_hiddens`).